src/style.css

```
@@ -1,3 +1,76 @@
-body {
+html {
+  --blue: #00acdc;
+}
+
+body,
+ul,
+a {
   margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.header {
+  position: relative;
+  display: flex;
+  font-family: Roboto, sans-serif;
+  justify-content: flex-end;
```

```suggestion
  justify-content: space-between;
```
just add here the space-between